enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +735,56 @@
+                                              :dataset_query {:type     :query
+                                                              :database db-id
+                                                              :query    {:source-table $$orders
+                                                                         :aggregation  [:sum [:field %orders.id nil]]
+                                                                         :breakout     [[:field %orders.user_id nil]]}}
+                                              :visualization_settings
+                                              {:pivot_table.column_split
+                                               {:rows    [[:field %people.name {:base-type    :type/Text
+                                                                                :source-field %orders.user_id}]]
+                                                :columns [[:field %products.title {:base-type    :type/Text
+                                                                                   :source-field %orders.product_id}]]
+                                                :values  [[:aggregation 0]]}
+                                               :column_settings
+                                               {(format "[\"ref\",[\"field\",%s,null]]" %people.name)
+                                                {:pivot_table.column_sort_order "descending"}}}}]
+                  (reset! card1s card)
+                  (-> (serdes/with-cache (into [] (extract/extract {})))
+                      (storage/store! dump-dir))))))
+
+          (ts/with-dest-db
+            (mt/dataset sample-dataset
+              (mt/$ids nil
+                (serdes/with-cache (serdes.load/load-metabase! (ingest/ingest-yaml dump-dir)))
+
+                (let [viz (t2/select-one-fn :visualization_settings Card :entity_id (:entity_id @card1s))]
+                  (is (contains? viz :pivot_table.column_split))
+                  ;; column ids are different in different dbs
+                  (is (not= (get-in @card1s [:visualization_settings :pivot_table.column_split :rows 0 1])
+                            (get-in viz [:pivot_table.column_split :rows 0 1])))
+                  (is (= [[:field %people.name {:base-type    :type/Text
+                                                :source-field %orders.user_id}]]
+                         (get-in viz [:pivot_table.column_split :rows])))
+                  (is (= [[:field %products.title {:base-type    :type/Text
+                                                   :source-field %orders.product_id}]]
+                         (get-in viz [:pivot_table.column_split :columns])))
+                  (is (= "descending"
+                         (get-in viz [:column_settings
+                                      (format "[\"ref\",[\"field\",%s,null]]" %people.name)
+                                      :pivot_table.column_sort_order]))))))))))))
```

Comment:
I'd create a `testing` block for this, maybe even referencing the old issue `(metabase#30134)`